docs/pages/colorschemeprovider.js

```diff
@@ -77,6 +60,7 @@ function Example(props) {
           />
         </MainSection.Subsection>
       </MainSection>
+
       <MainSection name="Related">
```

Comment:
@AlbertCarreras is this section still relevant? Is this holdover content from when the providers were combined into one?

Reply:
This is not longer relevant. Should be removed.